app/graphql/concerns/custom_payload_type.rb

```diff
@@ -0,0 +1,35 @@
+module CustomPayloadType
+  extend ActiveSupport::Concern
+
+  CRUD_OPERATION = %w[create update delete].freeze
+
+  class_methods do
+    # NOTE: Hook Override - https://github.com/rmosolgo/graphql-ruby/issues/2737
+    def generate_payload_type
+      resolver_name = generate_resolver_name
+      resolver_fields = fields
+
+      Class.new(object_class) do
+        graphql_name("#{resolver_name}Payload")
+        description("Autogenerated return type of #{resolver_name}")
+
+        field :errors, [Errors::UserError],
```

Comment:
I think I just need to check if the resolver_fields includes an errors field and only create if nothing there. This will allow for overrides per mutation level and they can use different error types.